platform/integrations/rancher/install-rancher-integration.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Install the Rancher integration in vCluster
+sidebar_label: Rancher Integration
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'
+import Flow, { Step } from '@site/src/components/Flow'
+
+# Install the Rancher integration
+
+The vCluster Rancher integration uses an operator to connect virtual clusters to Rancher by creating a corresponding Rancher cluster resource for each virtual cluster. It runs in the local cluster and handles registration automatically. Rancher users do not need cluster management permissions to use this integration. The operator assigns Rancher cluster owner access to the virtual cluster creator, project owners, and project members. This setup allows virtual clusters to appear and behave like managed clusters in Rancher.
+
+This guide explains how to install the vCluster Rancher Operator.
+
+## Prerequisites
+
+Ensure you have the following prerequisites before installing the vCluster Rancher Operator:
+
+- Access to the Rancher UI with admin or cluster-manager permissions
```

Comment:
Access to rancher UI is redundant. If they are admins or cluster-managers, they are users of rancher and have UI access. Let's combine this with the charts requirement.
```suggestion
- Admin permissions for the "local" cluster. You must be able to add Helm repositories and install charts in the "local" cluster.
```